homeassistant/components/jvc_projector/const.py

```diff
@@ -1,5 +1,6 @@
 """Constants for the jvc_projector integration."""
 
+
```

Comment:
Please revert

Reply:
@joostlek sorry do not understand what I have to do, what you mean with revert? there should be no change here.

Reply:
There's an extra line. Sorry for the nitpicking but let's try to clean this PR as much as possible :)

Reply:
understood